forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/CrmDelegate.java

```diff
@@ -146,20 +156,30 @@ private void crmOperation(DelegateExecution execution) {
         System.out.println("Finished CRM operation");
     }
 
-    private CrmPostResponse createCrmIncident(int contactId, DelegateExecution execution) {
+    private CrmPostResponse createCrmIncident(int managerContactId, int employeeContactId, DelegateExecution execution) {
         // Loading the CRM fields from the form
         String crmProductLookupName = String.valueOf(execution.getVariables().get(CRM_MAT_PAT_PRODUCT_LOOKUP_NAME_FIELD));
         String crmCategoryLookupName = String.valueOf(execution.getVariables().get(CRM_MAT_PAT_CATEGORY_LOOKUP_NAME_FIELD));
         String crmStaffGroupLookupName = String.valueOf(execution.getVariables().get(CRM_MAT_PAT_STAFF_GROUP_LOOKUP_NAME_FIELD));
         String crmSubject = String.valueOf(execution.getVariables().get(CRM_MAT_PAT_SUBJECT_FIELD));
         String submitterDisplayName = String.valueOf(execution.getVariables().get(CRM_MAT_PAT_SUBMITTER_NAME_FIELD));
         String threadText = String.valueOf(execution.getVariables().get(CRM_THREAD_TEXT_FIELD));
-        
-        String crmIncidentSubject = crmSubject + " for " + submitterDisplayName;
-        CrmPrimaryContact crmPrimaryContact = new CrmPrimaryContact(contactId);
         if (threadText == null) {
             threadText = "";
         }
+        
+        String crmIncidentSubject = crmSubject + " for " + submitterDisplayName;
+        CrmPrimaryContact crmPrimaryContact = new CrmPrimaryContact(managerContactId);
+        CrmOtherContact crmOtherContact = new CrmOtherContact(employeeContactId);
+        ArrayList<CrmOtherContact> crmOtherContacts = new ArrayList<CrmOtherContact>();
+        crmOtherContacts.add(crmOtherContact);
+        // If the manager and employee are same, then set crmOtherContacts to null.
+        // @TODO: Revisit this logic before going to production
```

Comment:
For now, allowing a user to be both employee and manager for easier testing. Revisit this before going to production.